op-node/rollup/derive/l1_block_info.go

```diff
@@ -45,68 +45,112 @@ type L1BlockInfo struct {
 	L1FeeScalar   eth.Bytes32
 }
 
+//+---------+--------------------------+
+//| Bytes   | Field                    |
+//+---------+--------------------------+
+//| 4       | Function signature       |
+//| 24      | Padding for Number       |
```

Comment:
These padding + number pairs are a good candidate to be pulled out into a single read/write function (like `readSolidityABIUint64`)